raft/raft.go

```diff
@@ -596,6 +608,11 @@ func stepLeader(r *raft, m pb.Message) {
 			// drop any new proposals.
 			return
 		}
+		if r.leadTransferee != None {
```

Comment:
It's fine with me to drop proposals in this case (just like we already drop proposals on candidates or followers when `r.lead` is `None`).